devito/function.py

```diff
@@ -452,40 +453,44 @@ def _initialize_derivatives(self):
                     property(dx, 'Return the symbolic expression for '
                              'the centered first derivative wrt. '
                              'the %s dimension' % dim.name))
-
+            self.derivatives += ["d%s" % dim.name]
```

Comment:
I suggest to leave a blank line as before to neatly separate different code blocks

Reply:
ok